packages/driver/cross-origin-testing.md

```diff
@@ -6,8 +6,9 @@ The goal of this document is to give a technical overview of the architecture be
 
 See [Node.js’s URL doc](https://nodejs.org/api/url.html#url-strings-and-url-objects) for a handy breakdown of URL parts
 
-- **domain**: A hostname without the subdomain (for the purposes of this doc). May also be referred to as **superdomain** (e.g. `example.com`, `example.co.uk`, `localhost`)
-- **origin**: The combination of the protocol, hostname, and port of a URL. For the purposes of Cypress, the subdomain is irrelevant. (e.g. `http://example.com:3500`)
+- **domain**: The hostname portion of a URL. (e.g. `example.com`, `www.example.com`, `www.example.co.uk`, `localhost`)
+- **superdomain**: A domain without the subdomain. (e.g. `example.com`, `example.co.uk`, `localhost`)
```

Comment:
why is super domain the term without the subdomain? Seems like it would include it 😅 is this typically the root domain then?

Reply:
🤷‍♂️ I didn't come up with the term, so I don't really know. It's what we've used elsewhere in the codebase and the docs, so I thought it would be good to define it. I don't believe it's a widely-used term but one of our Cypress-isms like "multi-domain". I would guess it's because, like a superclass, it's a more general version of a domain, whereas including the subdomain is more specific. I do like "root domain" though. That's probably a better term for it.